### DISCUSSION TITLE
When I use python to install milvus-langchain ，the result said :  "No the module named pwd"

### DISCUSSION THREAD

**Discussion by AZNo0** [milvus-io/milvus#30719] [Q&A and General discussion]
![QQ截图20240221145738](https://github.com/milvus-io/milvus/assets/141487627/de531eb1-cb8d-4d83-a2a8-e6165d9cfbfd)

**Comment by yhmo** [comment]
Seems is a langchain issue: https://github.com/langchain-ai/langchain/issues/17514

"This issue is caused by pwd library, which is not available in windows."
